src/sentry/mediators/external_issues/creator.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.mediators import Mediator, Param
+from sentry.models import PlatformExternalIssue
+from sentry.utils.cache import memoize
+
+
+class Creator(Mediator):
```

Comment:
Is there a use-case for this to be used on its own? If not I would inline it into the the `IssueLinkCreator` personally.

Reply:
hmmm not at the moment but in the future would we @mnoble ?

Reply:
Probably not at the moment, product-wise, but I think it's partially worth it for composability's sake. We can always pull it out later if that proves necessary, though. Especially since it doesn't really contain any logic atm,